libdbus-sys/cross_compile.md

```diff
@@ -13,6 +13,22 @@ The examples below all assume you're trying to compile for Raspberry Pi 2 or 3 r
 Cross compiling using rust-embedded/cross
 =========================================
 
+#### New Instructions:
+
+**Update**: With the new `vendored` feature. It is no longer required to provide a custom docker image, or have `libdbus-1-dev` installed to build. Simply enable the `vendored` feature:
```

Comment:
Change to something like: 

It is not required to provide a custom docker image, or have `libdbus-1-dev` installed to build. Git will need to be installed on the build host in order to download the dbus sources. Then enable the `vendored` feature: